src/main/java/JavaLessons/Lesson1/Lesson3.java

```diff
@@ -0,0 +1,222 @@
+    public static void array100() {
+        System.out.print("int[] arr = {");
+        int[] arr = new int[100];
+        for (int i = 0; i < arr.length; i++) {
+           arr[i] = i + 1;
+           System.out.print(arr[i]);
+            if (i < arr.length -1) {
+                System.out.print(", ");
+            }
+        }
+        System.out.print("}\n");
+    }
+// Задать массив [ 1, 5, 3, 2, 11, 4, 5, 2, 4, 8, 9, 1 ] пройти по нему циклом, и числа меньшие 6 умножить на 2;
+public static void multiplyUnderSix() {
+    System.out.print("int[] arr = {");
+    int[] arr = {1, 5, 3, 2, 11, 4, 5, 2, 4, 8, 9, 1};
+    for (int i = 0; i < arr.length; i++) {
+        arr[i] = (arr[i] < 6) ? arr[i] * 2 : arr[i];
+        System.out.print(arr[i]);
+        if (i < arr.length -1) {
+            System.out.print(", ");
+        }
+    }
+    System.out.print("}\n");
+}
+public static void multiplyUnderSixV2() {
+        System.out.print("int[] arr = {");
+        int[] arr = {1, 5, 3, 2, 11, 4, 5, 2, 4, 8, 9, 1};
+        for (int i = 0; i < arr.length; i++) {
+            if (arr[i] < 6) {
+                arr[i] = arr[i] * 2;
+            }
+            System.out.print(arr[i]);
+            if (i < arr.length -1) {
+                System.out.print(", ");
+            }
+        }
+        System.out.print("}\n");
+    }
+// Создать квадратный двумерный целочисленный массив (количество строк и столбцов одинаковое), и с помощью цикла(-ов) заполнить его диагональные элементы единицами (можно только одну из диагоналей, если обе сложно). Определить элементы одной из диагоналей можно по следующему принципу: индексы таких элементов равны, то есть [0][0], [1][1], [2][2], …, [n][n];
+public static void crossArray() {
+    int[][] arr = new int[5][5];
+    for (int i = 0; i < arr.length; i++) {
+      for (int j = 0; j < arr.length; j++) {
+          if ((i == j) || (i == arr.length - 1 - j)) {
+              arr[i][j] = 1;
+          }
+          System.out.print(arr[i][j] + " ");
+      }
+        System.out.println();
+    }
+}
+// Написать метод, принимающий на вход два аргумента: len и initialValue, и возвращающий одномерный массив типа int длиной len, каждая ячейка которого равна initialValue;
+public static void lenArray(int len, int initialValue) {
+        int[] arr = new int[len];
+        for (int i = 0; i < arr.length; i++) {
+          arr[i] = initialValue;
+            System.out.print(arr[i] + " ");
+        }
+    System.out.println();
+}
+// Задать одномерный массив и найти в нем минимальный и максимальный элементы ;
+    public static int[] minMaxArray(int length) {
+        int[] arr = new int[length];
+        Random random = new Random();
+        for (int i = 0; i < arr.length; i++) {
+            arr[i] = random.nextInt();
+        }
+        Arrays.sort(arr);
+        System.out.println(Arrays.toString(arr));
+        return arr;
+    }
```

Comment:
that's a strange method. It is called "minMaxArray" and creates a sorted array with pseudorandom numbers. I mean I don't see how the name represents the purpose.